Vinyl/Vinyl.swift

```diff
@@ -8,11 +8,44 @@
 
 import Foundation
 
-final class Vinyl {
+struct Vinyl {
     
     let tracks: [Track]
     
     init(plastic: Plastic) {
         tracks = plastic.map(Track.init)
     }
+    
+    init(tracks: [Track]) {
+        
+        self.tracks = tracks
+    }
 }
+
+struct VinylFactory {
+    
```

Comment:
And we would be great having a mechanism to create multiple tracks per vinyl instead of a single one, it may be frequent having tests where there are more than one request being performed.